test-source/test/ceylon/time/testDuration.ceylon

```diff
@@ -14,3 +14,24 @@ import ceylon.time {
 parameters (`value scalableDurationTests`)
 shared test void testScalableDuration(Duration expectedDuration, Integer scale)
         => assertEquals { expected = expectedDuration; actual = scale ** Duration(1000); };
+
+shared test void testPlus()
```

Comment:
Truth to be told, I’d much prefer named arguments list here — that would make the expectation and actual value much more clearer.

Reply:
Ok, I'll change that later. (Just pushed changes for @CPColin 's second comment, the first one was too subtle for me :-) )

Reply:
Done in https://github.com/eclipse/ceylon-sdk/pull/714/commits/b1dd1be0b8e03bbde09f8a59fb74b4a2a1bb1773